src/driver/sample_experimental_design.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2021 The Private Cardinality Estimation Framework Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A sample experimental design."""
+
+from absl import app
+from absl import flags
+import itertools
+import math
+import numpy as np
+from pyDOE import lhs
+from typing import Iterable
+
+from wfa_planning_evaluation_framework.driver.experiment_parameters import (
+    ExperimentParameters,
+)
+from wfa_planning_evaluation_framework.driver.modeling_strategy_descriptor import (
+    ModelingStrategyDescriptor,
+)
+from wfa_planning_evaluation_framework.simulator.modeling_strategy import (
+    ModelingStrategy,
+)
+from wfa_planning_evaluation_framework.simulator.privacy_tracker import (
+    PrivacyBudget,
+)
+from wfa_planning_evaluation_framework.simulator.system_parameters import (
+    LiquidLegionsParameters,
+    SystemParameters,
+)
+from wfa_planning_evaluation_framework.driver.trial_descriptor import (
+    TrialDescriptor,
+)
+
+MODELING_STRATEGIES = [
+    ModelingStrategyDescriptor(
+        "m3proposal", {}, "goerg", {}, "restricted_pairwise_union", {}
```

Comment:
Just to check m3 proposal. According to [this doc](https://docs.google.com/document/d/19sa6OSbwomq2eZ9ihfdjJE9QS5hmSVWB8x-Hp7BD2YU/edit?resourcekey=0-pke6Xbnjz8P0oXkFigC9Qw#heading=h.4j2f5fwxawf7), m3 proposal considers single-pub model =gamma-poisson and multi-pub model=restricted_pairwise or generalized_mixture. Since generalized_mixture is not implemented yet, we only consider gamma-poisson and restricted_pairwise for now.

So, goerg is still not an option for M3 proposal, right?

Reply:
You are right that Goerg's model was not included in the original M3 proposal, but testing Goerg's model as part of the M3 proposal does not require any additional effort, so I thought it best to include it.  WDYT?

Reply:
SGTM!